src/types/Account/Account.d.ts

```diff
@@ -98,8 +98,12 @@ export declare namespace Account {
     }
 
     interface createTransactionOptions {
-        recipient?: RecipientOptions,
+        recipient?: PublicAddress,
+        satohis?: number,
+        amount?: number,
+
         recipients?: [RecipientOptions]
+        
```

Comment:
This is in order to improve readability between single recipient case (the three first elements), recipients cases, and the rest of the available options.